src/Microsoft.DotNet.Build.Tasks/PackageFiles/executor/executor.cs

```
@@ -0,0 +1,227 @@
+            {
+                return properties[valueToFind]["default"];
+            }
+            return null;
+        }
+
+        private string findPropertyType(string valueToFind)
+        {
+            if (properties.ContainsKey(valueToFind))
+            {
+                return properties[valueToFind]["type"];
+            }
+            return null;
+        }
+
+        public string buildCommand(Command commandToExecute, string os)
+        {
+            string completeCommand = string.Empty;
+            if(processes.ContainsKey(commandToExecute.processName))
+            {
+                completeCommand = (os.Equals("windows")) ? processes[commandToExecute.processName].run["windows"] : processes[commandToExecute.processName].run["non-windows"];
+                completeCommand += string.Format(" {0}", commandToExecute.project);
+                completeCommand += buildPropertiesForCommand(commandToExecute.property, commandToExecute.processName);
+            }
+            else
+            {
+                Console.WriteLine("Error: The process {0} is not specified in the Json file.", commandToExecute.processName);
+            }
+            
+            return completeCommand;
+        }
+
+        public string buildPropertiesForCommand(Dictionary<string, string> propertiesToParse, string processName)
+        {
+            string commandProperties = string.Empty;
+            foreach(KeyValuePair<string,string> prop in propertiesToParse)
+            {
+                var value = (prop.Value.Equals(string.Empty)) ? findPropertyValue(prop.Key): prop.Value;
+                if (value!=null)
+                {
+                    commandProperties += string.Format(" {0}",formatProperty(prop.Key, value, findPropertyType(prop.Key), processName));
+                }
+                else
+                {
+                    Console.WriteLine("Error: The property {0} is not specified in the Json file.", prop.Key);
```

Shouldn't this probably abort the command? This error is probably going to get swallowed up in the impending output from a full build or the like.